src/routes/create/+page.server.js

```diff
@@ -1,5 +1,44 @@
 import { db } from '$lib/server/db';
 import { redirect } from '@sveltejs/kit';
+import { z } from 'zod';
+
+const filterSchema = z.object({
+	subjectArea: z
+		.string({ required_error: 'Ein Fachbereich wird benötigt' })
+		.min(1, { message: 'Ein Fachbereich wird benötigt' })
+		.trim(),
+	areaOfExpertise: z
+		.string({ required_error: 'Ein Fachgebiet wird benötigt' })
+		.min(1, { message: 'Ein Fachgebiet wird benötigt' })
+		.trim(),
+	specialization: z
```

Comment:
In der Maske steht Spezialisierung, nicht Spezifikation. Aber wir wollten das ja in Keywords umbenennen oder? Also kannst du auch direkt Keyword schreiben denke ich.

Reply:
Ja können wir unter der Woche für den Code Review in allen Dateien ändern.